packages/core/src/Prelude/Filterable/index.ts

```diff
@@ -0,0 +1,75 @@
+import type { Either } from "../../Classic/Either"
+import type { Option } from "../../Classic/Option"
+import type { Predicate, Refinement } from "../../Function"
+import type { Separated } from "../../Utils"
+import type { Compactable } from "../Compactable"
+import type { Covariant } from "../Covariant"
+import type * as HKT from "../HKT"
+
+export interface Filterable<F extends HKT.URIS, C = HKT.Auto>
```

Comment:
Filterable should be a `type` that intersect its childs like `Filterable = Covariant & Filter & FilterMap & Partition`